PinpointKit/PinpointKit/Sources/Configuration.swift

```diff
@@ -60,7 +60,7 @@ public struct Configuration {
                 feedbackCollector: FeedbackCollector = FeedbackNavigationController(),
                 editor: Editor = EditImageViewController(),
                 sender: Sender = MailSender(),
-                feedbackRecipients: [String]? = nil) {
+                feedbackRecipients: [String]) {
```

Comment:
We don’t have a stored property here. It’s a `var` on `FeedbackCollector`. Since our conformer is `FeedbackNavigationController`, and for better compatibility with IB, I think it’d be safer to stay optional. Let me know if you disagree.